Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14-buster-slim
```

Comment:
Can we use node 16? Node 14 is in maintenance mode.

Reply:
I will try

Reply:
all test pass with 16, will use it.